stellar-multitenant/pkg/tenant/manager.go

```diff
@@ -292,12 +292,7 @@ func (m *Manager) UpdateTenantConfig(ctx context.Context, tu *TenantUpdate) (*Te
 		args = append(args, *tu.Status)
 	}
 
-	if tu.DistributionAccount != nil {
-		fields = append(fields, "distribution_account = ?")
-		args = append(args, *tu.DistributionAccount)
-
-		log.Ctx(ctx).Warnf("distribution account for tenant id %s updated to %s", tu.ID, *tu.DistributionAccount)
-	}
+	fields, args = m.updateDistributionAccountFields(ctx, tu, fields, args)
```

Comment:
Just to confirm, this update is used by the `Post`, but isn't used by `Patch` tenants as we're not updating the patch fields.

Reply:
That's correct, we had concluded that distribution account types should not be updated, right? So they can only be set during the tenant provisioning process.

The work to activate a Circle account will require these two tickets to be in place:

1. [\[SDP-1167\] When provisioning a tenant, indicate the signer type](https://stellarorg.atlassian.net/browse/SDP-1167)
2. [\[SDP-1170\] \[BE\] We need the ability to PATCH the API Key and walletID (Circle keys expire every so often - 1 year ?)](https://stellarorg.atlassian.net/browse/SDP-1170)